client/src/locales/en-US.json

```diff
@@ -68,13 +67,7 @@
         "noInvitations": "No invitations"
     },
     "notification": {
-        "changeRole": "Your role a been changed to {role} on the workspace {workspace}",
-        "workspaceAccess": "You have access to the workspace {workspace}",
-        "userJoinWorkspace": "{name} has joined the workspace {workspace}",
-        "multipleUsersJoinWorkspace": "{count} have joined the workspace {workspace}",
```

Comment:
`git grep "multipleUsersJoinWorkspace"`

```
src/components/notifications/MultipleUsersJoinWorkspaceNotification.vue:          keypath="notification.multipleUsersJoinWorkspace"
src/locales/en-US.json:        "multipleUsersJoinWorkspace": "{count} have joined the workspace {workspace}",
src/locales/fr-FR.json:        "multipleUsersJoinWorkspace": "{count} viennent de rejoindre l'espace de travail {workspace}",
```
Why was it removed? There's probably a few others like this.